instance/tasks.py

```diff
@@ -145,37 +169,34 @@ def spawn_appserver(
     )
 
     if appserver and mark_active_on_success:
-        make_appserver_active(appserver, active=True, deactivate_others=deactivate_old_appservers)
-
-    # Huey doesn't seem to resolve properly if the result of a task is so return False
```

Comment:
@fox, is this still an issue? If yes, we should return `False`.

Reply:
We're no longer running this task in a map() function and doing a result check, so it shouldn't be a problem.